### DISCUSSION TITLE
move dotfiles directory into ~/.config directory

### DISCUSSION THREAD

**Discussion by Bammede** [mylinuxforwork/dotfiles#59] [Ideas]
I wonder why dotfiles files aren't inside .config directory?

**Comment by mylinuxforwork** [comment]
The installer creates symbolic links from the dotfiles folder into the .config folder. This setup gives me the chance to control the git repository much easier. It is a common setup for managing dotfiles. 

A great tool to generate dynamic links for configurations is stow.

Currently I have implemented the creation of symlinks by myself but planning to replace this with stow.

**Reply by Bammede** [reply]
If it is used just for GIT, can I remove the "dotfiles" directory and those symlinks will be removed automatically?

P.S. I am going yo use https://www.chezmoi.io/ - it is better alternative to Stow.

**Reply by mylinuxforwork** [reply]
No. If you want to uninstall the dotfiles please run uninstall from the welcome app. This will remove the dotfiles folder and all symlinks.

was not aware of chezmoi. Interesting.

**Reply by Bammede** [reply]
I want to use your dotfiles, but do I need to keep "dotfiles" directory? Is "dotfiles" dir a copy of settings from ~/.config files related dotfiles? You have mentioned you use "dotfiles" to manage GIT, I will not because I gonna use chezmoi.

**Reply by mylinuxforwork** [reply]
I am planning to clean up and rearrange the files for 2.9.5. That could also make it easier to use chezmoi to manage the files.
